File: frontend/src/types/paper.ts
import type { PageQuery } from './common'

/** 纸张档案，与后端 Paper 对应（含 BaseEntity 通用字段，按需取用）。 */
export interface Paper {
  uuid: string
  paperCode?: string
  paperName: string
  /** 常用克重 g/㎡ */
  gramWeight?: number
  paperType?: string
  remark?: string
  createTime?: string
  updateTime?: string
}

/** 纸张新增/修改入参，与后端 PaperSaveDTO 对应。 */
export interface PaperSaveDTO {
  paperCode?: string
  paperName: string
  gramWeight?: number
  paperType?: string
  remark?: string
}

/** 纸张列表查询入参。 */
export interface PaperQuery extends PageQuery {
  keyword?: string
}

File: src/main/java/com/paper/mes/paper/entity/Paper.java
package com.paper.mes.paper.entity;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.TableName;
import com.paper.mes.common.BaseEntity;
import lombok.Data;
import lombok.EqualsAndHashCode;

/**
 * 纸张档案 sys_paper。
 */
@Data
@EqualsAndHashCode(callSuper = true)
@TableName("sys_paper")
public class Paper extends BaseEntity {

    @TableId(type = IdType.ASSIGN_UUID)
    private String uuid;

    private String paperCode;
    private String paperName;
    /** 常用克重 g/㎡ */
    private Integer gramWeight;
    private String paperType;
    private String remark;
}
